json-db.json

```diff
@@ -512,6 +512,46 @@
       "hasReserved": false,
       "reservations": [],
       "maxRequestPriority": -2147483648
+    },
+    {
+      "applicationID": "application-sleep-0002",
```

Comment:
There is already an app-0002 in the database.
For distinguishing the applications, replace 0002 with 0006.

Reply:
Updated it. Thank you.